### PR TITLE
docs(accessibility): Accessibility contributing guide

### PR DESCRIPTION
**Pull request by jurokapsiar**
This PR adds more information about accessibility in the Stardust contributing guide

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -58,35 +58,43 @@ These guides will walk your through various activities for contributing:
 
 ## Accessibility
 
+Stardust implements accessibility using accessibility behaviors. The behaviors add attributes to the DOM elements (mainly role and aria-* properties) as well as handle keyboard interaction and focus. Every accessible component has a defiault behavior, which can be overriden using the `accessibility` prop.
```
Comment (miroslavstastny) [inline .github/CONTRIBUTING.md]: defiault -> default

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -58,35 +58,43 @@ These guides will walk your through various activities for contributing:
...
+
+Behaviors apply properties, focus handling and keyboard handlers to the component slots. When developing a component, the properties and keyboard handlers need to be spreaded to the corresponding slots.
```
Comment (miroslavstastny) [inline .github/CONTRIBUTING.md]: spreaded -> spread

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -58,35 +58,43 @@ These guides will walk your through various activities for contributing:
...
+
+For complete accessibility contributing guide, requireements and testing, see [Accessibility][2]
```
Comment (miroslavstastny) [inline .github/CONTRIBUTING.md]: requireements -> requirements

**Review by miroslavstastny — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -58,35 +58,43 @@ These guides will walk your through various activities for contributing:
 
 ## Accessibility
 
+Stardust implements accessibility using accessibility behaviors. The behaviors add attributes to the DOM elements (mainly role and aria-* properties) as well as handle keyboard interaction and focus. Every accessible component has a defiault behavior, which can be overriden using the `accessibility` prop.
+
```
Comment (smykhailov) [inline .github/CONTRIBUTING.md:62]: I'd probably add here something like: 
To override behavior, you can use the existing one from the behavior catalog or implement your own if none is suitable for the component.

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -58,35 +58,43 @@ These guides will walk your through various activities for contributing:
...
+- [Keyboard handling](#keyboard-handling)
 
 ### Role and aria props
 
-ARIA landmarks are attributes you can add to elements in your page to define areas like the main content or a navigation region.
-It is used to help the screen readers to find the correct elements on the page, where the semantic HTML is not able to cover
-it (like for elements of type pop-up).
-
-It is represented through the `role` and `aria-*` attributes which are usually added to divs and spans.
+ARIA [roles](https://www.w3.org/TR/wai-aria-1.1/#usage_intro) and [attributes](https://www.w3.org/TR/wai-aria-1.1/#introstates) provide information for screen readers to allow the users to navigate in the page/application.
```
Comment (kuzhelov) [inline .github/CONTRIBUTING.md]: seems that we may be more general here, as this screen readers doesn't cover the full set of technologies that consume ARIA attributes/roles. Something like

> provide necessary semantics for assistive technologies that allow persons with disabilities to to navigate in the page/application.

**Review by kuzhelov — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 5 above)
Comment (alinais) [reply .github/CONTRIBUTING.md]: please add references to all the urls at the end of the file

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -58,35 +58,43 @@ These guides will walk your through various activities for contributing:
...
 
-Stardust provides already the needed ARIA attributes to make its components accessible.
+In addition to behaviors, ARIA [landmarks](https://www.w3.org/TR/wai-aria-1.1/#landmark_roles) and [naming props](https://www.w3.org/TR/wai-aria-1.1/#namecalculation) need to be added to the components/elements to form the page structure and provide textual information.
```
Comment (alinais) [inline .github/CONTRIBUTING.md]: also please add this url as reference at the end of the file

**Review by alinais — CHANGES_REQUESTED** [review]

**Review by alinais — APPROVED** [review]

**Pull request merged**